### PR TITLE
Add readme text

### LINKED ISSUE #55: Amend the README to give "getting started" instructions

### PR DESCRIPTION
**Pull request by rbalekai**
closes #55

### CODE REVIEW

**Comment by rfox2** [comment]
I'm good with this as a first pass :+1:

File: README.rdoc [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
+Getting Started:
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Requirements:
 
-Things you may want to cover:
+* Ruby (~> 2.1.4)
+* RabbitMQ (~3.5.0) - https://www.rabbitmq.com/download.html
+* Postgres - (~ 9) - http://www.postgresql.org/download/
```
Comment (jaronkk) [inline README.rdoc]: We're making use of the Postgres jsonb datatype, so we should specify Postgres needs to be >= 9.4

**Comment by jaronkk** [comment]
@rbalekai Can you rename it from README.rdoc to README.md so we get the Markdown formatting?

**Comment by jgondron** [comment]
:+1:

**Pull request merged**